rero_ils/modules/loans/api.py

```diff
@@ -555,6 +555,24 @@ def get_any_loans_by_item_pid_by_patron_pid(item_pid, patron_pid):
         item_pid, patron_pid, filter_states)
 
 
+def count_any_loans_by_patron_pid(patron_pid):
+    """Count loans not ITEM_IN_TRANSIT_TO_HOUSE, CREATED for patron.
```

Comment:
I cannot understand. :slightly_smiling_face: 

Is it *Count loans that are not ITEM_IN_TRANSIT_TO_HOUSE but CREATED for a given patron.*?